manim/utils/space_ops.py

```diff
@@ -235,9 +344,20 @@ def angle_of_vector(vector: Sequence[float]) -> float:
 
 
 def angle_between_vectors(v1: Sequence[float], v2: Sequence[float]) -> float:
-    """
-    Returns the angle between two 3D vectors.
+    """Returns the angle between two 3D vectors.
```

Comment:
```suggestion
    """Returns the angle between two vectors. 
```
As this also works for the typical 2 D vectors, mentioning 3D here can be a bit misleading